api/apps/articles/model/save.coffee

```diff
@@ -342,9 +343,15 @@ typecastIds = (article) ->
     debug err if err
     cb()
 
-@deleteArticleFromSailthru = (article, cb) =>
+cleanArticlesInSailthru = (slugs = []) =>
+  if slugs.length > 1
+    _.map slugs, (slug, i) =>
```

Comment:
Minor: Not sure if `_.map` is needed here because you could just use a loop or `slugs.forEach`.